src/main/scala/nl/knaw/dans/easy/stage/Conf.scala

```diff
@@ -67,12 +67,16 @@ class Conf(args: Seq[String]) extends ScallopConf(args) {
     default = Some(false))
   val stageFileDataAsRedirectDatastreams = opt[Boolean](
     name = "stage-file-data-as-redirect-datastreams", short = 'r',
-    descr = """Stage file items so that the content of file data will NOT be stored in managed Fedora Storage""",
+    descr = "Stage file items so that the content of file data will NOT be stored in managed Fedora Storage",
     default = Some(false))
   val fileDataRedirectBaseUrl = opt[URL](
     name = "file-data-redirect-base-url", short = 'b',
-    descr = """Base URL from which redirect URLs to the file items in this dataset will be constructed""",
+    descr = "Base URL from which redirect URLs to the file items in this dataset will be constructed",
     default = None)
+  val stubAVfiles = opt[Boolean](
+    name = "stub-av-files", short = 's',
+    descr = "Stage audio or video (AV) file items so that the content of file data will NOT be stored in managed Fedora Storage",
```

Comment:
Another proces ends to do that, easy-ingest-flow perhaps?
Anyway, easy-stage-dataset is not responsible for that.